addons/website/static/src/snippets/s_embed_code/000.scss

```diff
@@ -2,3 +2,8 @@
 .editor_enable .s_embed_code {
     min-height: $o-font-size-base;
 }
+
+.s_embed_code .s_embed_code_embedded .bg-light.o_default_snippet_text {
+    color: black;
+}
```

Comment:
Would not this break if `light` is configured to be a dark color (since 17.0 (I think?) especially since light is a configurable color?)

Maybe it's because text-bg-light should be used instead?

Reply:
> Would not this break if `light` is configured to be a dark color (since 17.0 (I think?) especially since light is a configurable color?)
> 
> Maybe it's because text-bg-light should be used instead?

@qsm-odoo  Indeed, depending on the site's color configuration, `bg-light` could appear dark. However, thinking about it, I don’t believe such a technical element should be affected by the color palette. I pushed a change forcing the background to `lightcyan` (in Master, the XML might need to be adapted, perhaps with an alert and an icon). Also, I wonder if we should hide the element outside of edit mode. What’s your opinion?

Reply:
> I don’t believe such a technical element should be affected by the color palette.

@qsm-odoo Can I have your opinion on this ?

Reply:
@bvr-odoo I agree although it does not matter a lot to me :+1: 

> Also, I wonder if we should hide the element outside of edit mode. What’s your opinion?

Same, I think we probably can hide it with our special class which is designed for that (css_editable_mode_only, something like that) but it does not matter a lot. As long as it is readable, anything is fine by me.

Reply:
> @bvr-odoo I agree although it does not matter a lot to me 👍
> 
> > Also, I wonder if we should hide the element outside of edit mode. What’s your opinion?
> 
> Same, I think we probably can hide it with our special class which is designed for that (css_editable_mode_only, something like that) but it does not matter a lot. As long as it is readable, anything is fine by me.

@qsm-odoo ok thanks, let's leave it as it is. Could you review/merge the PR please